homeassistant/components/bring/config_flow.py

```diff
@@ -66,7 +66,7 @@ async def async_step_user(
                 await self.async_set_unique_id(bring.uuid)
                 self._abort_if_unique_id_configured()
                 return self.async_create_entry(
-                    title=user_input[CONF_EMAIL], data=user_input
+                    title=info["name"] or user_input[CONF_EMAIL], data=user_input
```

Comment:
why the `or`?

Reply:
The mobile app does not allow the name to be empty, but if the registration happend through the browser app it is possible that the name is empty, hence the fall-back to email.